public/usage-examples/graphics/fill_circle_on_bitmap/fill_circle_on_bitmap-1-rainbow-caterpillar-oop.cs

```diff
@@ -0,0 +1,45 @@
+using SplashKitSDK;
+
+namespace FillCircleOnBitmap
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a window and bitmap for our caterpillar
+            Window window = new Window("Window", 400, 200);
+            Bitmap bitmap = new Bitmap("caterpillar", 400, 200);
+
+            // Fill background with light color
+            bitmap.Clear(Color.White);
```

Comment:
try SplashKit.ClearBitmap(bitmap, Color.White); to keep it consistent with other languages